app/validators/blocks/grand_calculated_summary_block_validator.py

```diff
@@ -83,39 +87,80 @@ def validate_calculated_summary_is_before_grand_calculated_summary_block(
                     calculated_summary_id=calculated_summary_id,
                 )
 
-    def validate_repeating_calculated_summaries(self):
+    def validate_calculated_summaries(self):
         """
-        If the grand calculated summary references a repeating calculated summary, this is only valid if:
-        1) the grand calculated summary is also repeating
-        2) it is in the same repeating section as the repeating calculated summary it references
+        Run additional validation for the scenarios:
+        1) any grand calculated summary referencing a repeating calculated summary
+        2) repeating grand calculated summary referencing a static calculated summary
         """
-        gcs_section_id = self.questionnaire_schema.get_section_id_for_block_id(
+        gcs_section = self.questionnaire_schema.get_parent_section_for_block(
```

Comment:
Only minor, but quite a mixture of using cs/calculated_summary and gcs/grand_calculated_summary in these files, would be good to make this consistent.

Reply:
Yeah the variable names were just really long, but consistency is important I agree, have made them all full name